subprojects/docs/src/docs/userguide/build_init_plugin.adoc

```diff
@@ -13,50 +13,38 @@
 // limitations under the License.
 
 [[build_init_plugin]]
-== Build Init Plugin
+= Build Init Plugin
 
 
 [NOTE]
 ====
 
-The Build Init plugin is currently <<feature_lifecycle,incubating>>. Please be aware that the DSL and other configuration may change in later Gradle versions.
+The Build Init plugin is currently <<feature_lifecycle.adoc#feature_lifecycle,incubating>>. Please be aware that the DSL and other configuration may change in later Gradle versions.
 
 ====
 
 The Gradle Build Init plugin can be used to bootstrap the process of creating a new Gradle build. It supports creating brand new projects of different types as well as converting existing builds (e.g. An Apache Maven build) to be Gradle builds.
 
-Gradle plugins typically need to be _applied_ to a project before they can be used (see <<sec:using_plugins>>). The Build Init plugin is an automatically applied plugin, which means you do not need to apply it explicitly. To use the plugin, simply execute the task named `init` where you would like to create the Gradle build. There is no need to create a “stub” `build.gradle` file in order to apply the plugin.
+Gradle plugins typically need to be _applied_ to a project before they can be used (see <<plugins.adoc#sec:using_plugins,Using plugins>>). The Build Init plugin is an automatically applied plugin, which means you do not need to apply it explicitly. To use the plugin, simply execute the task named `init` where you would like to create the Gradle build. There is no need to create a “stub” `build.gradle` file in order to apply the plugin.
 
-It also leverages the `wrapper` task to <<sec:adding_wrapper,generate the Gradle Wrapper files>> for the project.
+It also leverages the `wrapper` task to <<gradle_wrapper.adoc#sec:adding_wrapper,generate the Gradle Wrapper files>> for the project.
 
 
 [[sec:build_init_tasks]]
-=== Tasks
+== Tasks
 
 The plugin adds the following tasks to the project:
 
-.Build Init plugin - tasks
-[cols="a,a,a,a", options="header"]
-|===
-| Task name
-| Depends on
-| Type
-| Description
-
-| `init`
-| `wrapper`
-| api:org.gradle.buildinit.tasks.InitBuild[]
-| Generates a Gradle project.
-
-| `wrapper`
-| -
-| api:org.gradle.api.tasks.wrapper.Wrapper[]
-| Generates Gradle wrapper files.
-|===
+`init` — type: link:{groovyDslPath}/org.gradle.buildinit.tasks.InitBuild.html[InitBuild]::
```

Comment:
Let's drop `type: `.

Reply:
Will do in a separate PR to avoid additional noise.

Reply:
👍